app/controllers/pg_hero/home_controller.rb

```diff
@@ -258,6 +258,13 @@ def explain
     def tune
       @title = "Tune"
       @settings = @database.settings
+      @citus_enabled = @database.citus_enabled?
+      if @citus_enabled
+        @citus_nodesno = @database.citus_nodesno
```

Comment:
You can omit assigning @database.citus_nodesno to a variable if you are not going to use it in other PRs.